lute/templates/read/reading_menu.html

```diff
@@ -123,6 +123,11 @@
             Mobile
           </a>
         </li>
+        <li>
+          <a id="screen-mobile-focus" class="reading-menu-item" href="" onclick="set_screen_type('mobile-focus'); return false;">
```

Comment:
I'm not sure about this -- the screen type is still `mobile`, you just happen to be changing something about how the interactions are done on mobile.  It feels like this should set some different kind of local storage setting.